### PR TITLE
Added HTML file

### PR DESCRIPTION
**Pull request by Beanalini**
Added html

### CODE REVIEW

**Review by munnos — COMMENTED** [review]
LGTM

**Review by JosephBalshaw — APPROVED** [review]

**Review by munnos — APPROVED** [review]
LGTM

**Review by AmaalSaid — APPROVED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
+</head>
+
+<body>
+    <div>
+        
+    </div>
```
Comment (AmaalSaid) [inline index.html:11]: change

Comment (JosephBalshaw) [inline index.html:11]: Lacks a class of the div so the CSS can be more easily applied

**Review by JosephBalshaw — COMMENTED** [review]
LGTM

**Pull request merged**